the-new-developers/gatehopper
# Refactor application to use ES6 modules

## Changes:
Closes #49 

Refactors the application to use ES6 modules as well as javascript classes while preserving existing functionality. (Please see related issue for more details).

## How to test this PR:
- Pull this branch locally
- Open index.html in your browser with some sort of server running - i.e. the Live Server or Live Preview extensions in VS Code, or a local server (this is so you can see the images)
- There should be no errors in the browser's console
- Clicking on each of the menu buttons should change the screen without generating any console errors
- Clicking on the "Cheat Sheet" menu button should display the cheat sheet and back button
- Clicking on the "Back" button in the cheat sheet screen should bring you back to the main menu

## Checklist:
